src/backend/CMakeLists.txt

```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 set(BACKEND_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(D3D12_DIR ${CMAKE_CURRENT_SOURCE_DIR}/d3d12)
 set(METAL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/metal)
 set(NULL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/null)
 set(OPENGL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/opengl)
-set(D3D12_DIR ${CMAKE_CURRENT_SOURCE_DIR}/d3d12)
-set(TESTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tests)
```

Comment:
What happened to this line?

Reply:
Tests aren't in src/backend anymore, it was unused.